.github/workflows/build-and-push-images.yaml

```diff
@@ -45,7 +46,7 @@ jobs:
         id: publish
         uses: ./.github/workflows/template-publish-image
         with:
-          image: docker.io/kubeflow/${{ matrix.component-name }}
+          image: ghcr.io/kubeflow/${{ matrix.component-name }}
```

Comment:
Does GHCR support additional paths so we can distinguish images by Kubeflow Project name:
```
docker pull ghcr.io/kubeflow/trainer/trainer-controller-manager
```

https://github.com/orgs/kubeflow/packages/container/package/training%2Ftraining-operator

Reply:
yes, you are right - done.